PolyEngine/CzolgInvaders/Src/InvadersGame.cpp

```diff
@@ -39,13 +40,31 @@ void InvadersGame::Init()
 
 void InvadersGame::Deinit()
 {
-	Engine->GetWorld().DestroyEntity(Camera);
+	HeavyTaskSystem::DestroyEntity(&Engine->GetWorld(), Camera);
 	for(auto ent : GameEntities)
-		Engine->GetWorld().DestroyEntity(ent);
+		HeavyTaskSystem::DestroyEntity(&Engine->GetWorld(), ent);
```

Comment:
Both these calls should be immediate, as the heavy task system won't tick after deinit.